stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/repository/helper/BaseRepositoryFacadeTest.kt

```diff
@@ -0,0 +1,55 @@
+package io.getstream.chat.android.livedata.repository.helper
+
+import androidx.annotation.CallSuper
+import com.nhaarman.mockitokotlin2.mock
+import io.getstream.chat.android.livedata.repository.RepositoryFacade
+import io.getstream.chat.android.livedata.repository.domain.channel.ChannelRepository
+import io.getstream.chat.android.livedata.repository.domain.channelconfig.ChannelConfigRepository
+import io.getstream.chat.android.livedata.repository.domain.message.MessageRepository
+import io.getstream.chat.android.livedata.repository.domain.queryChannels.QueryChannelsRepository
+import io.getstream.chat.android.livedata.repository.domain.reaction.ReactionRepository
+import io.getstream.chat.android.livedata.repository.domain.syncState.SyncStateRepository
+import io.getstream.chat.android.livedata.repository.domain.user.UserRepository
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineScope
+import org.junit.jupiter.api.BeforeEach
+
+@ExperimentalCoroutinesApi
+internal open class BaseRepositoryFacadeTest {
```

Comment:
Rename from `BaseRepositoryHelperTest` to `BaseRepositoryFacadeTest`. No new code